### PR TITLE
Add pryzm mainnet and update pryzm testnet

### PR DESCRIPTION
**Pull request by arthuryeti**
Hey guys :)

Let us know if something is wrong.

Thank you

### CODE REVIEW

**Comment by editaahn** [comment]
Hi! Since we merge PRs for registering Keplr-supported chains right before the launch time, if you need testnet changes to be merged, please separate PRs for testnet and mainnet changes. Then, I'll merge the testnet changes right away.

**Comment by arthuryeti** [comment]
@editaahn all good :) testnet can wait.

File: cosmos/pryzm.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
+  "rpc": "https://rpc.pryzm.zone",
+  "rest": "https://api.pryzm.zone",
+  "nodeProvider": {
+    "name": "PRYZM",
+    "email": "support@pryzm.zone",
+    "website": "https://pryzm.zone/"
+  },
+  "chainId": "pryzm-1",
+  "chainName": "pryzm",
```
Comment (editaahn) [inline cosmos/pryzm.json:10]: Is it intentional for the chain name to start with the lowercase letter?

Comment (editaahn) [reply cosmos/pryzm.json:10]: I'm just wondering whether it's fine for you!

**Comment by editaahn** [comment]
I write additional lines to your changes through the new PR below for the native chain support. 
https://github.com/chainapsis/keplr-chain-registry/pull/633
I'll close this PR and we can continue our conversation and the whole progress on the new PR.

**Pull request closed without merging**